types/markdown-it/v13/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "private": true,
+    "name": "@types/markdown-it",
+    "version": "13.0.9999",
+    "projects": [
+        "https://github.com/markdown-it/markdown-it"
+    ],
+    "dependencies": {
+        "@types/linkify-it": "^3",
```

Comment:
This should be ^4, given https://unpkg.com/markdown-it@13.0.2/package.json says linkify-it at 4.

Reply:
There are no types corresponding to v4 - https://www.npmjs.com/package/@types/linkify-it?activeTab=versions

Reply:
Bleh, okay. I guess the other PR jumped right to 5.

Reply:
v4 didn't had any type changes though - https://github.com/markdown-it/linkify-it/blob/master/CHANGELOG.md